crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -447,6 +474,21 @@ impl ReadOnlyUserIdentity {
         &self.self_signing_key
     }
 
+    /// Check if our identity is verified.
```

Comment:
This certainly doesn't mean that it is "verified", which would imply we have done the cross-signing dance.